tools/clang/test/DXC/Passes/DxilRemoveUnstructuredLoopExits/exit-via-false-propagate-exit-cond.ll

```diff
@@ -0,0 +1,83 @@
+; RUN: opt %s -analyze -loops | FileCheck -check-prefix=LOOPBEFORE %s
```

Comment:
I think we might want these tests to live in [test/HLSL/passes](https://github.com/microsoft/DirectXShaderCompiler/tree/main/test/HLSL/passes). The test organization is still unclear to me, but I think for `opt` tests we generally want them out of the `tools/clang` directory. 

I think the `clang/test/DXC/Passes` is more for things that either need `dxc` or `dxopt` to run correctly.

Reply:
I've moved the tests as suggested.